shared-helpers/src/locales/general.json

```diff
@@ -308,6 +308,10 @@
   "application.review.confirmation.whatHappensNext.lottery": "### What happens next?\n\n* Once the application period closes, eligible applicants will be placed in order based on lottery rank order.\n\n* Housing preferences, if applicable, will affect lottery rank order.\n\n* If you are contacted for an interview, you will be asked to fill out a more detailed application and provide supporting documents.",
   "application.review.confirmation.whatHappensNext.waitlist": "### What happens next?\n\n* Eligible applicants will be placed on the waitlist on a first come first serve basis until waitlist spots are filled.\n\n* Housing preferences, if applicable, will affect waitlist order.\n\n* If you are contacted for an interview, you will be asked to fill out a more detailed application and provide supporting documents.\n\n* You may be contacted while on the waitlist to confirm that you wish to remain on the waitlist.",
   "application.review.confirmation.whatHappensNext.base": "### What happens next?\n\n* If you are contacted for an interview, you will be asked to fill out a more detailed application and provide supporting documents.",
+  "application.review.confirmation.whatHappensNext.noPref.fcfs": "### What happens next?\n\n* Eligible applicants will be contacted on a first come first serve basis until vacancies are filled.\n\n* If you are contacted for an interview, you will be asked to fill out a more detailed application and provide supporting documents.",
```

Comment:
We need translations for these in all other languages, we can use google translate for now!